Core/GUIConfiguration.cs

```diff
@@ -3,6 +3,8 @@
 using System.IO;
```

Comment:
`GUIConfiguration.cs` has "GUI" in its name, it should not be moved from GUI to Core.